zerver/tests/test_audit_log.py

```diff
@@ -57,6 +57,8 @@ def test_change_role(self) -> None:
         do_change_user_role(user_profile, UserProfile.ROLE_MEMBER)
         do_change_user_role(user_profile, UserProfile.ROLE_GUEST)
         do_change_user_role(user_profile, UserProfile.ROLE_MEMBER)
+        do_change_user_role(user_profile, UserProfile.ROLE_REALM_OWNER)
+        do_change_user_role(user_profile, UserProfile.ROLE_MEMBER)
```

Comment:
This change should have also modified the check function.  I went ahead and added this and then merged this commit for simplicity; see 7d9d2414ca631bb6e31b369d03156e3a82cd8112.